Description: Add thumbnailer
 This patch adds a thumbnailer for the dia MIME type.
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/653469

Index: dia-0.97.3+git20160930/Makefile.am
===================================================================
--- dia-0.97.3+git20160930.orig/Makefile.am
+++ dia-0.97.3+git20160930/Makefile.am
@@ -93,6 +93,10 @@ dist-hook:
 		echo A git clone is required to generate a ChageLog >&2; \
 	fi
 
+install-data-local:
+	$(mkinstalldirs) $(DESTDIR)$(datadir)/thumbnailers
+	$(INSTALL_DATA) $(srcdir)/dia.thumbnailer $(DESTDIR)$(datadir)/thumbnailers/dia.thumbnailer
+
 if MAINTAINER_MODE
 THANKS: all-recursive
 	app/run_dia.sh --credits > THANKS
Index: dia-0.97.3+git20160930/dia.thumbnailer
===================================================================
--- /dev/null
+++ dia-0.97.3+git20160930/dia.thumbnailer
@@ -0,0 +1,4 @@
+[Thumbnailer Entry]
+TryExec=dia
+Exec=dia -t png -e %o -s %s %i
+MimeType=application/x-dia-diagram;
